src/enrichment_tables/geoip.rs

```diff
@@ -180,10 +182,17 @@ impl Geoip {
 
                 let location = data.location.as_ref();
                 add_field!("timezone", location.and_then(|location| location.time_zone));
-                add_field!("latitude", location.and_then(|location| location.latitude));
+                add_field!(
+                    "latitude",
+                    location
+                        .and_then(|location| location.latitude)
+                        .map(|latitude| Value::Float(NotNan::new(latitude).unwrap()))
```

Comment:
Removing the `test` feature reveals a 'hidden' unwrap call. See [here](https://github.com/vectordotdev/vrl/blob/a4f9c517b8db6fafae0b42b6db31096b633c5b15/src/value/value/convert.rs#L338).